src/Data/PQueue/Min.hs

```diff
@@ -263,6 +263,6 @@ keysF :: (pRk k a -> rk k) -> Prio.BinomForest pRk k a -> BinomForest rk k
 keysF f ts0 = case ts0 of
   Prio.Nil       -> Nil
   Prio.Skip ts'  -> Skip $! keysF f' ts'
-  Prio.Cons (Prio.BinomTree k _ ts) ts'
+  Prio.Cons (Prio.BinomTree k ts) ts'
     -> Cons (BinomTree k (f ts)) $! keysF f' ts'
-  where  f' (Prio.Succ (Prio.BinomTree k _ ts) tss) = Succ (BinomTree k (f ts)) (f tss)
+  where  f' (Prio.Succ (Prio.BinomTree k ts) tss) = Succ (BinomTree k (f ts)) (f tss)
```

Comment:
This function needs to be modified to drop values eagerly. Unfortunately, that requires writing a rank 0 wrapper around a general worker.

Reply:
Never mind. That's wrong. It does need to be modified, but the modification is trivial.

Reply:
Wrong again. I got the whole thing backwards; no modification is needed; it's already exactly as it needs to be.